packages/synapse-react-client/src/components/ChallengeDataDownload/ChallengeDataDownload.tsx

```diff
@@ -70,11 +71,18 @@ export function ChallengeDataDownload({
     isIdSelected,
     isSelectable: () => true,
     toggleSelection,
+    hiddenColumns: [
+      DetailsViewColumn.BADGES,
+      DetailsViewColumn.ADD_TO_DOWNLOAD_CART,
+      DetailsViewColumn.CREATED_ON,
+      DetailsViewColumn.VERSION,
+      DetailsViewColumn.MODIFIED_BY,
+    ],
```

Comment:
For the challenge portal version of this component, hide these columns (instead of maintaining tons of duplicate code!).